src/yang/ietf-bgp-rib-tables.yang

```diff
@@ -70,6 +69,45 @@ submodule ietf-bgp-rib-tables {
       "RFC XXXX, BGP YANG Model for Service Provider Network.";
   }
 
+  grouping bgp-common-route-annotations-state {
+    description
+      "Data definitions for flags and other information attached
+       to routes in both LOC-RIB and Adj-RIB";
+    leaf last-modified {
+      type yang:timeticks;
+      description
+        "Timestamp when this path was last modified.
+
+         The value is the timestamp in seconds relative to
+         the Unix Epoch (Jan 1, 1970 00:00:00 UTC).";
+    }
+    leaf valid-route {
+      type boolean;
+      description
```

Comment:
What exactly is "valid" meant to be referring to in the BGP spec sense of things?

Reply:
This might be completely Cisco thing, and Keyur can confirm. A route can be valid or invalid when the next-hop is unreachable. And then there is the whole discussion of what does it mean to validate a route in RFC 5123. But I am not an expert, so I will let the two of you tell me how you would rephrase this node, or remove it if it does not make sense.

Reply:
5123 isn't a good match here.
If the intent is meant to be "is the route eligible for route install", then we can craft some words from rfc 4271.  
I'll wait on Keyur to respond.

Reply:
This issue is still open.